com.github.imagefanreloaded.json

```diff
@@ -0,0 +1,37 @@
+{
+    "app-id" : "com.github.imagefanreloaded",
+    "runtime" : "org.freedesktop.Platform",
+    "runtime-version" : "22.08",
+    "sdk" : "org.freedesktop.Sdk",
+    "command" : "ImageFanReloaded-Avalonia",
+    "finish-args" : [
+		"--socket=x11",
+        "--share=ipc",
+        "--device=dri",
+		"--filesystem=host:ro"
+    ],
+    "modules" : [
+        {
+            "name": "ImageFanReloaded-Avalonia",
+            "buildsystem": "simple",
+            "sources": [
+				{
+					"type": "file",
+					"only-arches": [
+						"x86_64"
+					],
+					"url": "https://github.com/mihnea-radulescu/imagefanreloaded/releases/download/1.2023.08.28/ImageFanReloaded-Avalonia-Linux-x64-Flathub.zip",
```

Comment:
how does this work on the aarch64 flatpak that is built?

Reply:
Thank you for raising this, I've duly updated the manifest by adding a separate aarch64 build.